doc/reference/core-prelude.md

```diff
@@ -1014,7 +1014,7 @@ Defines the following symbols as externs:
     flmax flmin
     box? box unbox set-box!
     make-list cons*
-    foldl foldr andmap ormap filter filter-map iota last last-pair
+    foldl foldr andmap ormap everymap anymap filter filter-map iota last last-pair
```

Comment:
I'm not convinced by the names of these functions, or by the claim that they belong in the prelude as opposed to `std/misc/list`

Reply:
**Prelude or Not**
If passing multiple arguments to one predicate belongs to the prelude, the opposite belongs there probably too. I thought also about `std/misc/list`, but mapping one value over predicates and returning a `boolean` seems not so "listy". The alternative would be to move it to `std/misc/func`. How did you in the past decide when something should be moved to the prelude?

**Naming things is hard**
I know there are some similarities with `srfi-1` names. Using 'every' and 'any' felt naturally like the correct name for those procedures viewed from a Gerbil perspective in combination with `andmap` and `ormap` (they form a group). Naming is in this case quite hard, since there are so many things that have nearby concepts with already fixed meanings (other common synonyms are: all, each, sum, total).

Alternative names I came up with:
- `match-every`, `match-any`
- `everytrue`, `anytrue`
- `all-in`, `one-in`
- `every-of`, `any-of`

Reply:
I never put anything in the prelude, and would like to keep it minimal. I don't think these things appears often enough to be featured here. @vyzo's opinion is required.

As for naming, well... Is `(everymap fs v)` the same as `(every (cut <> v) fs)` ? I'm not even sure we need it in the stdlib, but maybe `every-pred` or something like that?

Reply:
Just moved the procedures to `misc/list` and renamed them to `every-of` & `any-of` and also changed the procedure signature to more closely match `andmap`.
If this is okay I would rebase and remove the other commit.

Reply:
That's good. In general, we don't put anything in core unless it's a primitive required by the system to bootstrap etc.